include/deal.II/grid/tria.h

```diff
@@ -40,6 +40,7 @@ DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
 #include <list>
 #include <map>
 #include <algorithm>
+#include <numeric> // need for std::accumulate
```

Comment:
Ah, I just added <algorithm> yesterday. Please remove <algorithm> again, and then also drop the comment.